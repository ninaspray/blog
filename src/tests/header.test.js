import React from "react";
import { render, screen } from "@testing-library/react";
import '@testing-library/jest-dom'
import Header from "../components/Header";

describe("Header", () => {
    it("contains title", () => {
      render(<Header />);
      expect(screen.getByText("Blog Demo")).toBeInTheDocument();
    });
    it("shows theme buttons", () => {
      const { container } = render(<Header />);
      expect(container.lastChild).toMatchSnapshot();
    })
  });