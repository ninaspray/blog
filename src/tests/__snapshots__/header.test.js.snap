// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header shows theme buttons 1`] = `
<header
  class="App-header"
>
  <h1>
    Blog Demo
  </h1>
  <div>
    <button
      type="button"
    >
      light
    </button>
    <button
      type="button"
    >
      dark
    </button>
  </div>
</header>
`;
